src/sig_stfl/sig_stfl.h

```diff
@@ -165,7 +165,7 @@ OQS_API int OQS_SIG_STFL_alg_is_enabled(const char *method_name);
  */
 typedef struct OQS_SIG_STFL {
 
-	/** A local ordinal representing the LMS parameter of the signature scheme. */
+	/** A local ordinal representing the OID parameter of the signature scheme. */
```

Comment:
Is there a place where I can read up what an "OID" is in this context? 32bits are surely not enough to represent an ASN.1 OID. When looking around, I found `sig_stfl_xmss.h`: If this header file is meant to document the externally usable API (?), it IMO needs quite a bit more documentation to be of help to any user. Please consider adding this before considering a merge to "main". Apologies for possibly jumping the gun here, but you asked me for review. Or was this unintended?

Reply:
@baentsch , you're right. Thanks for your review, I will update the comment. 
The intended idea of "OID" is it's only for the `external/`, it should not be understood or related to ASN.1 OID.

Reply:
Updated in https://github.com/open-quantum-safe/liboqs/pull/1588/commits/8e422a569c96a9d6254a0274d84b01ac3884af1e